addons/project/models/project.py

```diff
@@ -1178,6 +1182,13 @@ def _inverse_personal_stage_type_id(self):
         for task in self:
             task.personal_stage_id.stage_id = task.personal_stage_type_id
 
+    @api.depends('date_last_stage_update')
+    def _compute_days_from_last_stage_update(self):
+        for task in self:
+            task.days_from_last_stage_update = False
+            if task.date_last_stage_update:
+                task.days_from_last_stage_update = (fields.Datetime.now() - task.date_last_stage_update).days
+
```

Comment:
Can you move this compute (and the related search in the following suggestion) below the `_search_personal_stage_type_id` method, I prefer to keep the compute and search methods for the same field at the same place. 😇 

Here is the search method to avoid storing this field:
```py
    @api.model
    def _search_days_from_last_stage_update(self, operator, value):
        return [('days_last_stage_update', expression.TERM_OPERATORS_NEGATION[operator], fields.Datetime.now() - relativedelta(days=value))]  # FIXME: need to import: `from dateutil.relativedelta import relativedelta`
```

ps: I didn't test this code.